package.json

```diff
@@ -1,22 +1,18 @@
 {
-  "name": "chat-app",
-  "version": "1.0.0",
-  "description": "",
+  "name": "trivy-scans-test",
+  "version": "0.0.1",
+  "description": "Just a test",
   "main": "index.js",
   "scripts": {
-    "start": "node src/index.js",
-    "dev": "nodemon src/index.js"
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "",
+  "author": "lior",
   "license": "ISC",
+  "repository": "Some repo string",
   "dependencies": {
-    "bad-words": "^3.0.0",
-    "express": "^4.17.1",
-    "socket.io": "^2.4.0"
-  },
-  "devDependencies": {
-    "nodemon": "^1.18.7"
+    "express": "4.18.1",
+    "lodash": "4.17.21",
+    "react": "18.2.0",
+    "axios": "0.19.2"
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2020-28168   
**Check Name:** nodejs-axios: allows an attacker to bypass a proxy by providing a URL that responds with a redirect to a restricted host or IP address   
**Severity:** MEDIUM   
**Fixed Version:** 0.21.1   
**Description:** Axios NPM package 0.21.0 contains a Server-Side Request Forgery (SSRF) vulnerability where an attacker is able to bypass a proxy by providing a URL that responds with a redirect to a restricted host or IP address.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2020-28168